### PR TITLE
Update common_ci.properties 

### PR DESCRIPTION
**Pull request by sac2kadam**
## 📋 Description

JIRA ID: 

Please provide a summary of the change and the motivation behind it. Include relevant context and details.

---

## ✅ Type of Change

- [ ] 🐞 **Bug fix** (non-breaking change which resolves an issue)
- [ ] ✨ **New feature** (non-breaking change which adds functionality)
- [ ] 🔥 **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🛠 **Refactor** (change that is neither a fix nor a new feature)
- [ ] ⚙️ **Config change** (configuration file or build script updates)
- [ ] 📚 **Documentation** (updates to docs or readme)
- [ ] 🧪 **Tests** (adding new or updating existing tests)
- [ ] 🎨 **UI/UX** (changes that affect the user interface)
- [ ] 🚀 **Performance** (improves performance)
- [ ] 🧹 **Chore** (miscellaneous changes that don't modify src or test files)

---

## ℹ️ Additional Information

Please describe how the changes were tested, and include any relevant screenshots, logs, or other information that provides additional context.

## Summary by CodeRabbit

- **New Features**
	- Introduced multiple new configuration properties for enhanced service management, including identifiers for various services and authentication keys.
	- Added a new scheduling slot size property for improved scheduling capabilities.

- **Bug Fixes**
	- Updated the prescription property for correct definition and formatting.

- **Style**
	- Minor formatting adjustments for improved readability.

### CODE REVIEW

File: src/main/environment/common_ci.properties [hunk 1]
```diff
@@ -64,7 +64,17 @@ labWL=7
...
 tcSpeclistWL=7
-
+servicePointID: @env.SERVICE_POINT_ID@
+parkingPlaceID: @env.PARKING_PLACE_ID@
+providerServiceMapID: @env.PROVIDER_SERVICE_MAP_ID@
+vanID: @env.VAN_ID@
+serviceID: @env.SERVICE_ID@
+providerID: @env.PROVIDER_ID@
+appId: @env.APP_ID@
+authKey: @env.AUTH_KEY@
```
Comment (drtechie) [inline src/main/environment/common_ci.properties:74]: @sac2kadam can we add a context to this?
What is this AUTH keys about?
@ravishanigarapu can you help?

File: src/main/environment/common_ci.properties [hunk 2]
```diff
@@ -64,7 +64,17 @@ labWL=7
 radioWL=7
 oncoWL=7
 tcSpeclistWL=7
-
+servicePointID: @env.SERVICE_POINT_ID@
+parkingPlaceID: @env.PARKING_PLACE_ID@
+providerServiceMapID: @env.PROVIDER_SERVICE_MAP_ID@
+vanID: @env.VAN_ID@
+serviceID: @env.SERVICE_ID@
+providerID: @env.PROVIDER_ID@
```
[the remark this answers is not in the document]
Comment (drtechie) [reply src/main/environment/common_ci.properties:72]: @sac2kadam

File: src/main/environment/common_ci.properties (hunk 1 above)
Comment (sac2kadam) [reply src/main/environment/common_ci.properties:74]: @devikasuresh20 and   @srishtigrp78 
they are checking about AUTH key,

Comment (srishtigrp78) [reply src/main/environment/common_ci.properties:74]: As per my analysis till now, we are taking these properties from application.properties file to get nurse master data for beneficiary in code using state id, it is being used in PatientAppCommonMasterController.
@drtechie @sac2kadam

**Pull request merged**